### PR TITLE
data is transposed at the beginning of the functioon

### PR DESCRIPTION
**Pull request by resatomo**
[#178](https://github.com/aolabNeuro/analyze/issues/178)

### CODE REVIEW

**Review by leoscholl — COMMENTED** [review]
You’ll have to update the tests as well, the figures are broken now

File: aopy/precondition.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def butterworth_filter_data(data,fs, cutoff_freq= None, bands= None,  order= Non
     Apply a digital butterworth filter forward and backward to a timeseries signal.
 
     Args:
-        data (array): neural data (n_channels x n_samples)
+        data (array): neural data (n_samples x n_channels)
```
Comment (leoscholl) [inline aopy/precondition.py:48]: Can you move the size of the array into the parentheses? See the docstring writing guide https://analyze.readthedocs.io/en/latest/documentation-guidelines.html

**Comment by resatomo** [comment]
`plot_db_spectral_estimate` takes log, but frequency power of test data contains 0 at various frequencies. So, I plotted PSD through welch and multitaper without taking log.

**Comment by leoscholl** [comment]
did some tidying up. just swapping the axes around didn't really work, i don't think the code had ever been tested on multichannel data.

**Review by leoscholl — APPROVED** [review]

**Pull request merged**